scripts/test/compareBenchmarkScriptTest.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
```

Comment:
doc what this file is good for? Having read it once, I couldn't describe what is actually tested here.

Reply:
```
#!/usr/bin/env python3

import os
import re
import json
import math
import subprocess

# This script takes two JSON result files, runs the compare_benchmarks.py script, and checks that the output is as
# expected.
```

Ok?